blacklight-cornell/app/models/concerns/blacklight/document/endnote_xml.rb

```diff
@@ -229,7 +227,10 @@ def generate_enx_dates(bld,ty)
   def generate_enx_contributors(bld,ty)
     authors = get_all_authors(to_marc)
     relators =  get_contrib_roles(to_marc)
```

Comment:
I think this should be looked into further. `relators`  seems to be used by this one logging method call, but there may be more under the hood going on with the `get_contrib_roles`.

Reply:
Created a ticket for this for future cleanup:  Jira ticket- https://culibrary.atlassian.net/browse/DACCESS-766